### PR TITLE
feat(optimus): add backup detail rpc

### PR DESCRIPTION
**Pull request by arinda-arif**
- Adding configuration field in backup spec
- Add backup detail rpc

### CODE REVIEW

**Comment by sravankorumilli** [comment]
Looks good.

File: odpf/optimus/runtime_service.proto [hunk 1]
```diff
@@ -221,6 +221,11 @@ service RuntimeService {
             get: "/v1/project/{project_name}/namespace/{namespace}/datastore/{datastore_name}/backup"
         };
     }
+    rpc GetBackupDetail(GetBackupDetailRequest) returns (GetBackupDetailResponse) {
```
Comment (kushsharma) [inline odpf/optimus/runtime_service.proto:224]: We can call this just `GetBackup` right?

Comment (arinda-arif) [reply odpf/optimus/runtime_service.proto]: updated to GetBackup

**Comment by arinda-arif** [comment]
Also refactored Backup to CreateBackup @kushsharma

**Pull request merged**